# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/HTMLEditor.h
# Commit: 420e02d37505
# Full Hash: 420e02d3750587b6dfaa72229a38e4ff8bb38149
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-10-19 04:09:55
# Regressor Bug: 1778565
# File Overlap Count: 1
# Description:
#   Bug 1778565 - Editor: Handle nsIFile transferable as blob. r=nika,masayuki
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155753
# ==============================================================================

diff -r 3f19cab95424 -r 420e02d37505 editor/libeditor/HTMLEditor.h
--- a/editor/libeditor/HTMLEditor.h	Tue Oct 18 17:35:29 2022 +0000
+++ b/editor/libeditor/HTMLEditor.h	Tue Oct 18 17:35:33 2022 +0000
@@ -4261,10 +4261,10 @@
    * operation is finished.
    *
    * @param aBlob       The input blob
-   * @param aWindow     The global object under which the read should happen.
+   * @param aGlobal     The global object under which the read should happen.
    * @param aBlobReader The blob reader object to be notified when finished.
    */
-  static nsresult SlurpBlob(dom::Blob* aBlob, nsPIDOMWindowOuter* aWindow,
+  static nsresult SlurpBlob(dom::Blob* aBlob, nsIGlobalObject* aGlobal,
                             BlobReader* aBlobReader);
 
   /**